representing-data/representing-data-in-lists-and-dictionaries.md

```diff
@@ -37,7 +37,7 @@ Lists are great at...
 | Lists have a property of length/size      | A frequently used property for lists is its length. An empty list has a length of zero.                                             | We can use the `len` function and pass in a list to get its length.                                                  |
 | We can access elements in a list by index | If we know the index of an element in a list, we can get the element                                                                | We index with square brackets, `my_list[0]`                                                                          |
 | We can add things to lists                | We can add elements to lists                                                                                                        | `my_list.append(new_element)` will add this element to the end of the list                                           |
-| We can remove things to lists             | We can remove elements from lists                                                                                                   | `my_list.remove(value_to_remove)` will find the first instance of this value and remove it                   |
+| We can remove things from lists             | We can remove elements from lists                                                                                                   | `my_list.remove(value_to_remove)` will find the first instance of this value and remove it                   |
```

Comment:
The change looks fine, though I have to gripe that we show adding to the end (not arbitrary insertion) then show aribtrary removal (by value, not even by position, which we're trying to emphasize the importance of order/position in lists). Seriously, `remove` is almost always the wrong thing to do with a list.